webapp/service/medlemService.ts

```diff
@@ -0,0 +1,56 @@
+const API_URL = 'http://localhost:8000'
```

Comment:
localhost?

Reply:
Thanks for the reminder, I'll switch it too api.ntnui.no before merging 😁